### PR TITLE
integrate with usage data collection

### PR DESCRIPTION
**Pull request by reyang**
use the pre-release version of BI package and see if things would work end-to-end.
after this sprint's release, we'll need to switch to the official NPM BI package

### CODE REVIEW

File: src/devdisco.js [hunk 1]
```diff
@@ -28,6 +29,10 @@ var WifiApTransport = require('./lib/wifi-ap-transport');
 var VERSION = require('../package.json').version;
 
 function main(argv) {
+  bi.trackEvent('command_line_arguments', {
```
Comment (xscript) [inline src/devdisco.js:32]: > command_line_arguments [](start = 17, length = 22)

maybe add some unique prefix for DevDisco? Event name will be mixed up in backend.

**Comment by xscript** [comment]
:shipit:

File: package.json [hunk 2]
```diff
@@ -25,12 +25,13 @@
...
   },
   "dependencies": {
+    "az-iot-bi-test": "0.0.3",
     "dns": "^0.2.2",
     "mdns-js": "^0.5.0",
     "node-arp": "^1.0.5",
     "node-wifiscanner": "^0.2.1",
-    "vow": "^0.4.12",
-    "serialport": "^4.0.1"
+    "serialport": "^4.0.1",
```
Comment (donzh) [inline package.json:33]: Why do you change the order of these two packages? Is it necessary？

File: src/devdisco.js (hunk 1 above)
Comment (aovechki) [reply src/devdisco.js:32]: in general, should establish a naming convention for these

---

In reply to: [78687734](https://github.com/Azure/device-discovery-cli/pull/6#discussion_r78687734) [](ancestors = 78687734)

**Pull request merged**